ansible/roles/collectd/vars/main.yml

```diff
@@ -8,4 +8,4 @@ service_list_oracle: "oracle-ohasd oracleasm"
 
 # service lists vars
 service_list_linux: "chronyd sshd"
-service_list_weblogic: "weblogic-node-manager" # weblogic-server might also be needed to be added here
+service_list_weblogic: "weblogic-node-manager weblogic-healthcheck" # weblogic-server might also be needed to be added here
```

Comment:
I don't think we need weblogic-node-manager in this list.  Just weblogic-healthcheck will be enough.